AsyncAwait.Task1.CancellationTokens/Calculator.cs

```diff
@@ -1,21 +1,28 @@
 ﻿using System.Threading;
+using System.Threading.Tasks;
 
 namespace AsyncAwait.Task1.CancellationTokens;
 
 internal static class Calculator
 {
     // todo: change this method to support cancellation token
-    public static long Calculate(int n /*, CancellationToken token*/)
+    public static async Task<long> Calculate(int n, CancellationToken token)
```

Comment:
Are you sure you need to make this specific method as async and return `Task<long>`? Asynchronous machinery was created mostly for IO-bound operations, not CPU-bound.

Reply:
No need to change it to async (KISS). Changed the function to synchoronous.